src/odemis/gui/comp/viewport.py

```diff
@@ -781,17 +783,108 @@ class AngularResolvedViewport(ViewPort):
 
     # Default class
     canvas_class = miccanvas.AngularResolvedCanvas
-    bottom_legend_class = None
+    bottom_legend_class = RadioLegend
+
+    def __init__(self, *args, **kwargs):
+        super(AngularResolvedViewport, self).__init__(*args, **kwargs)
+        # TODO why not sufficient to place here?
+        # bottom_legend_class = RadioLegend
 
     def setView(self, view, tab_data):
+        """
+        Set the microscope view that this viewport is displaying/representing
+        *Important*: Should be called only once, at initialisation.
+
+        :param view:(model.View)
+        :param tab_data: (model.MicroscopyGUIData)
+        """
         assert(self._view is None)
 
         self._view = view
         self._tab_data_model = tab_data
+        self.stream = None
```

Comment:
Add a comment with the type of this variable (as `None` is not very helpful). Eg: `# current Stream displayed (or None)`.